configure

```diff
@@ -1262,6 +1259,10 @@ case "${ARCH}" in
         CFLAGS="${CFLAGS} ${floatabi}"
         SFLAGS="${SFLAGS} ${floatabi}"
 
+        if test $without_optimizations -eq 0; then
+            check_neon_intrinsics
+        fi
```

Comment:
We should only check if NEON intrinsics are available if the processor might support NEON or VFP. There is a lot of ARM processors that support neither. We currently support VFPv3 and VFPv4 when compiling with gcc or Visual C++.

Reply:
I think that would have to be an improvement for another PR. I don't even own a ARM machine.